guides/2022-02-22-aws-ec2.mdx

```diff
@@ -1,7 +1,9 @@
 ---
-title: "Deploying to AWS EC2"
-sidebar_label: "AWS EC2"
-sidebar_position: 2
+title: "Deploying Conduit to an AWS EC2 instance"
+slug: aws-ec2
+tag: ['EC2 Deploy']
+description: Learn how to deploy Conduit to an AWS EC2 instance
+keywords: ['deploy', 'aws', 'ec2', 'conduit']
 ---
 
 # Deploying to Amazon EC2
```

Comment:
Sidenote: the instructions for this are now simpler. It's about creating an installation dir manually, adding the binary to systemd, etc. Conduit now provides deb files, so the installation is simpler: https://github.com/ConduitIO/conduit?tab=readme-ov-file#debian

Reply:
I'll revisit this in another PR.

Reply:
Sounds good to me!